src/scipp/compat/__init__.py

```diff
@@ -4,6 +4,10 @@
 # @author Jan-Lukas Wynen
 
 from .pandas_compat import from_pandas
-from .xarray_compat import from_xarray, from_xarray_dataarray, from_xarray_dataset
+from .xarray_compat import (from_xarray, to_xarray, from_xarray_dataarray,
+                            from_xarray_dataset)
 
-__all__ = ['from_pandas', 'from_xarray', 'from_xarray_dataarray', 'from_xarray_dataset']
+__all__ = [
+    'from_pandas', 'from_xarray', 'to_xarray', 'from_xarray_dataarray',
+    'from_xarray_dataset'
```

Comment:
Not your change, but I wonder why the `from_xarray_dataarray` and `from_xarray_dataset` functions are here?

Reply:
I don't know, but I'll remove them I think. It says in the docstring
```
If you know in advance what type of object you need to convert, you can
also call :func:`from_xarray_dataarray` or :func:`from_xarray_dataset directly`.
```
but I will just remove.